### PR TITLE
Update Learning-Environment-Create-New.md

### PR DESCRIPTION
**Pull request by MedhaviMonish**
In the "Final Editor Setup" , I think their should be a Step to add Decision Parameters Script and it says Decision Period from 1 to 20.
Without this their was no action taken by the RollerAgent. After adding this step it worked for me.

### CODE REVIEW

File: docs/Learning-Environment-Create-New.md [hunk 1]
```diff
@@ -343,11 +343,12 @@ with our Agent code.
 
 1. Select the **RollerAgent** GameObject to show its properties in the Inspector
     window.
-2. Change **Decision Interval** from `1` to `10`.
-3. Drag the Target GameObject from the Hierarchy window to the RollerAgent
+2. Add the Decision Requester Parameters script with the Add Component button from the RollerAgent Inspector.
```
Comment (chriselion) [inline docs/Learning-Environment-Create-New.md]:
```suggestion
2. Add the Decision Requester script with the Add Component button from the RollerAgent Inspector.
```

File: docs/Learning-Environment-Create-New.md [hunk 2]
```diff
@@ -343,11 +343,12 @@ with our Agent code.
...
+3. Change **Decision Period** from `1` to `20`.
```
Comment (chriselion) [inline docs/Learning-Environment-Create-New.md]:
```suggestion
3. Change **Decision Period** from `1` to `10`.
```
The old docs said 10. Maybe 20 works too though?

**Comment by chriselion** [comment]
@MedhaviMonish Can you either apply those changes, or give me permission to? https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

**Comment by chriselion** [comment]
Also, you need to sign the CLA before we can merge your changes.

File: docs/Learning-Environment-Create-New.md (hunk 2 above)
Comment (MedhaviMonish) [reply docs/Learning-Environment-Create-New.md]: Screenshot (19)

It says Decision period can be Max 20 . Maybe 10 was earlier. And I tried to change it to 50 but it again changed back to 20.

**Comment by MedhaviMonish** [comment]
> Also, you need to sign the CLA before we can merge your changes.

This CLA seems to be for Unity Employees. I am just a student , do i have to sign it?

**Comment by NuclearDev0** [comment]
If tou can understand what are they talking so, Yes!

**Comment by chriselion** [comment]
> This CLA seems to be for Unity Employees. I am just a student , do i have to sign it?

I don't think I've ever been able to read the whole thing without falling asleep either :) Anybody that wants to contribute a pull request to the repo has to sign in. I think the first section here gives a good explanation of what it's there for: https://www.clahub.com/pages/why_cla

File: docs/Learning-Environment-Create-New.md (hunk 2 above)
Comment (chriselion) [reply docs/Learning-Environment-Create-New.md]: OK, I think I understand the confusion; it's not describing the range, it's describing the value you should set. I think this is a better wording:
```suggestion
3. Change **Decision Period** to `20`.
```

**Comment by chriselion** [comment]
Thanks for helping out with fixing this!

**Pull request merged**